parameter_files/ProjectParameters_PA2024CH.yml

```diff
@@ -0,0 +1,66 @@
+default:
+
+    paths:
+        data_location_ext: ../pacta-data/2023Q4/
+
+    reporting:
+        project_report_name: PA2024CH
+        display_currency: CHF
+        currency_exchange_value: 1.190405
+
+    parameters:
+        start_year: 2023
+        horizon_year: 5
+        select_scenario: WEO2023_NZE_2050
+        scenario_other: GECO2023_1.5C
```

Comment:
Of note:
`geco_2023` for `1.5C` only covers "other" sectors Aviation and Steel 
``` r
[R] > geco_2023 |> filter(scenario == "1.5C") |> distinct(sector)
# A tibble: 5 × 1
  sector  
  <chr>   
1 Aviation
2 Coal    
3 Oil&Gas 
4 Power   
5 Steel 
```

whereas `weo_2023` for `NZE_2050` covers "other" sectors Aviation, Cement and Steel 
``` r
[R] > weo_2023 |> filter(scenario == "NZE_2050") |> distinct(sector)
# A tibble: 7 × 1
  sector    
  <chr>     
1 Automotive
2 Aviation  
3 Coal      
4 Oil&Gas   
5 Power     
6 Steel     
7 Cement 
```

I imagine it would probably make sense to set it as `WEO2023_NZE_2050` for both?

Reply:
@jdhoffa @Antoine-Lalechere and @cjyetman all agree on this, so I'm pretty sure @NAYRA-HERRERA and Nick will too

changed `scenario_other: GECO2023_1.5C` to `scenario_other: WEO2023_NZE_2050`